### DISCUSSION TITLE
Can I Give Permission By Model Id?

### DISCUSSION THREAD

**Discussion by reasecret** [spatie/laravel-permission#2389] [Q&A] [answered]
I've a Group model and I want to allow only group members create posts. Let's say a user joined X group and this user can create a post on this group. But this user shouldn't create a post on Y group.

So is there a usage something like this?: `$user->givePermissionTo('create post', $group->id)`

**Comment by drbyte** [comment, marked as answer]
You can use Laravel's model policies feature for that.

**Reply by reasecret** [reply]
Thanks. That's right. I just wondered the package has a feature like this.
